protocol/x/clob/memclob/memclob.go

```diff
@@ -694,6 +694,7 @@ func (m *MemClobPriceTimePriority) DeleverageSubaccount(
 	subaccountId satypes.SubaccountId,
 	perpetualId uint32,
 	deltaQuantums *big.Int,
+	isFinalSettlement bool,
```

Comment:
suggestion: update func comment to note if `isFinalSettlement` is true it will be the oracle price? Also maybe note that error will be returned if the subaccount can't be deleveraged?